### PR TITLE
fix: incorrect export

### PR DESCRIPTION
**Pull request by GmBodhi**

### CODE REVIEW

**Comment by RedGuy12** [comment]
There are now 3 PRs for the same thing: #14, #15, and #20, not to mention https://github.com/mrlapizgithub/repl.it-db/pull/10 upstream. @kochman please fix this!

**Comment by GmBodhi** [comment]
I don't have hope for this. I defined my own types

**Comment by wbobeirne** [comment]
Was just bit by this helping out a bud who couldn't figure out why he was getting runtime errors. Any of the PRs @RedGuy12  are suitable fixes for this, any chance we could get this fixed? cc @brenoafb @jnnnthnn seems like y'all have contributed recently to this repo.

**Review by rayhanadev — APPROVED** [review]
LGTM

File: index.d.ts [hunk 1]
```diff
@@ -13,3 +13,5 @@ export class Client {
   public setAll(obj: Record<any, any>): Client;
   public deleteMultiple(...args: string[]): Client;
 }
+
+export default Client
```
Comment (cdmistman) [inline index.d.ts:17]: i'm getting this error:

> Top-level declarations in .d.ts files must start with either a 'declare' or 'export' modifier.ts(1046)

**Comment by cdmistman** [comment]
closing in favor #46

**Pull request closed without merging**